src/main/java/mops/services/CSVService.java

```diff
@@ -0,0 +1,38 @@
+package mops.services;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import org.springframework.stereotype.Service;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CSVService {
+
+    public static List<String[]> readFromCSV(String csvName){
+        List<String[]> help = new ArrayList<>();
+        try {
+            CSVReader csvReader = new CSVReader(new FileReader(csvName));
+            help = csvReader.readAll();
+            csvReader.close();
+        } catch (IOException | CsvException e) {
+            e.printStackTrace();
+        }
+        return help;
```

Comment:
Können wir help in result oder etwas anderes Aussagekräftigeres umbenennen?

Reply:
mach ich